build/crds/generate-crd-docs.sh

```diff
@@ -5,19 +5,24 @@ GEN_CRD_API_REFERENCE_DOC_VERSION=v0.3.0
 SCRIPT_CHECK_DOCS_DIFF=$(git diff --name-only --diff-filter=M | grep -Ec '/types\.go$')
 
 install_generator() {
-    go install github.com/ahmetb/gen-crd-api-reference-docs@${GEN_CRD_API_REFERENCE_DOC_VERSION}
+  go install github.com/ahmetb/gen-crd-api-reference-docs@${GEN_CRD_API_REFERENCE_DOC_VERSION}
 }
 
 run_gen() {
-  if [[  "$SCRIPT_CHECK_DOCS_DIFF" -gt 0 || "$1" = "--force" ]]; then
+  if [[ "$SCRIPT_CHECK_DOCS_DIFF" -gt 0 || "$1" = "--force" ]]; then
     echo "differences found in types.go, rebuilding specification.md"
-    install_generator
-    gen-crd-api-reference-docs \
-        -config="${SCRIPT_ROOT}/crd-docs-config.json" \
+    if [ -z "$UNFORCE_GEN_CRD_DOCS" ]; then
```

Comment:
I don't understand what it means to "unforce" something. :) 
The behavior I would expect is that we always build these crd docs when `make crds` is run. So I think the new env var would be `FORCE_GEN_CRD_DOCS` and also be checked on line 12 so we can trigger this always with `make crds`.

Reply:
Update it to `SKIP_GEN_CRDS_DOCS` which make sense I guess,

Otherwise we need to check for two positive cases for `build docs` and `build crds` cmd